### PR TITLE
Convert ProfileInfoCard Component to Material UI 

### PR DESCRIPTION
**Pull request by piyumaldk**
## Description
Basic objective was to implement the ProfileInfoCard using Material-UI

## Related Issue
#125 _Convert ProfileInfoCard to Material UI_ 

## Motivation and Context
- Removed ANTD from the ProfileInfoCard component 
- Used Material-UI for the front end development
- Created a separate file in helper for themes
- Git ignored firebase config file after talking with @thisuraseniya 
- Added react-image-crop for cropping 1:1 images
- Added react-image for image loading
- Improved the previously developed VIewProfile component
- Removed ANTD from the Profile component 

## How Has This Been Tested?
- Manually tested the components 

## Screenshots or GIF (In case of UI changes):

**Basic UI**
basic
**Image Cropping**
crop
**Image Loading**
loading
**Image Upload**
upload

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by Niweera — APPROVED** [review]

File: src/components/Profile/ProfileInfoCard/index.js [hunk 1]
```diff
@@ -1,256 +1,428 @@
...
-  LinkOutlined,
-  LinkedinFilled,
-  SettingOutlined,
-  FlagOutlined
-} from "@ant-design/icons";
+import React, { useState, useCallback, useRef, useEffect } from "react";
+import ReactCrop from "react-image-crop";
+import "react-image-crop/dist/ReactCrop.css";
+import { Img } from "react-image";
+import noImageAvailable from "../../../assets/images/no-image-available.svg";
```
Comment (shehand) [inline src/components/Profile/ProfileInfoCard/index.js:5]: Better to declare this in a separate config file and import the image through that.

File: src/components/Profile/ProfileInfoCard/index.js [hunk 2]
```diff
@@ -1,256 +1,428 @@
...
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import Chip from "@material-ui/core/Chip";
+import Dialog from "@material-ui/core/Dialog";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Divider from "@material-ui/core/Divider";
+import LinearProgress from "@material-ui/core/LinearProgress";
+import CircularProgress from "@material-ui/core/CircularProgress";
+
```
Comment (shehand) [inline src/components/Profile/ProfileInfoCard/index.js:20]: remove new line

File: src/config/index.js [hunk 3]
```diff
@@ -1,41 +0,0 @@
-import firebase from "firebase/app";
```
Comment (shehand) [inline src/config/index.js:1]: I think this will delete the file in this repo. Thought it was only ignoring. Do whatever @thisuraseniya asked you to do.

**Review by shehand — CHANGES_REQUESTED** [review]

File: src/config/index.js (hunk 3 above)
Comment (piyumaldk) [reply src/config/index.js]: Sorry. I will add the file and gitignore it.

File: src/components/Profile/ProfileInfoCard/index.js (hunk 2 above)
Comment (piyumaldk) [reply src/components/Profile/ProfileInfoCard/index.js]: I will remove the line

File: src/components/Profile/ProfileInfoCard/index.js (hunk 1 above)
Comment (piyumaldk) [reply src/components/Profile/ProfileInfoCard/index.js]: Thank you for mentioning that. I implemented a file called **images.js** in the helper folder _(The folder we use for general components)_ with an exporting image component and _svg_ files. We will add other _svg_ files when we are changing other components too in the future. 
@thisuraseniya @shehand Is it okay?

**Review by shehand — APPROVED** [review]

**Pull request merged**